validation/jinja2/validation/_wrong_buttons.html

```diff
@@ -0,0 +1,42 @@
+{#
+      Copyright (C) 2021 Jolene Poulin <jcpoulin@ualberta.ca>
+
+      This program is free software: you can redistribute it and/or modify
+      it under the terms of the GNU Affero General Public License as
+      published by the Free Software Foundation, either version 3 of the
+      License, or (at your option) any later version.
+
+      This program is distributed in the hope that it will be useful,
+      but WITHOUT ANY WARRANTY; without even the implied warranty of
+      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+      GNU Affero General Public License for more details.
+
+      You should have received a copy of the GNU Affero General Public License
+      along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#}
+
+{% macro button_col(recording, csrf_token, request) %}
+
+    {% if recording.quality == "wrong word" %}
+        <button class="button button--neutral wrong-speaker-button" data-cy="wrong-button" data-toggle="modal" data-target="#{{ recording.id }}-modal" data-rec-id={{ recording.id }}>Wrong Speaker</button>
+        <button class="button button--neutral-solid wrong-word-button" data-cy="wrong-button" data-rec-id={{ recording.id }} onclick="showWrongWordDiv('{{ recording.id }}')">Wrong Word</button>
+    {% elif recording.quality == "wrong speaker" %}
+        <button class="button button--neutral-solid wrong-speaker-button" data-cy="wrong-button" data-toggle="modal" data-target="#{{ recording.id }}-modal" data-rec-id={{ recording.id }}>Wrong Speaker</button>
+        <button class="button button--neutral wrong-word-button" data-cy="wrong-button" data-rec-id={{ recording.id }} onclick="showWrongWordDiv('{{ recording.id }}')">Wrong Word</button>
+    {% else %}
```

Comment:
What if a recording has both wrong word _and_ the wrong speaker?

Reply:
Excellent point. The UI would just show the most recent change and the DB would store that same most recent change. BUT there would be an Issue filed for both cases, so anyone with admin access can go in and see that the recording has both the wrong speaker code and the wrong word. In the future, I would consider making a linguist-specific interface for viewing and resolving these issues.